### PR TITLE
Add initial orga README

### PR DESCRIPTION
**Pull request by erikbosch**
If everything works as expected the file `profile/README.md` will become visible as README at https://github.com/eclipse-kuksa

I did a quick work to specify all our repos and status, it is most easily inspected if you go to https://github.com/erikbosch/.github/blob/erik_readme/profile/README.md

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+|   Repository        | Status         | Content/Comment
+|-----------------------|----------------|---------------------|
+| [kuksa.val](https://github.com/eclipse/kuksa.val)| Alpha/Deprecated | Feeders/Providers
+| [kuksa.val.feeders](https://github.com/eclipse/kuksa.val.feeders)| Production/Mature/Deprecated | KUKSA Databroker, KUKSA Server,KUKSA Go Client and more
+| [kuksa.val.services](https://github.com/eclipse/kuksa.val.services)| Alpha | Examples services using KUKSA
+
+
+### Archived Repositories
+
```
Comment (SebastianSchildt) [inline profile/README.md:105]: I would not mention anything archived in the "Old" Eclipse orga here

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+| [kuksa-databroker](https://github.com/eclipse-kuksa/kuksa-databroker)| Planning | KUKSA Databroker. Migration ongoing, content still part of [kuksa.val](https://github.com/eclipse/kuksa.val)
+
+
+### General Use Clients and SDKs
+
+
+|   Repository        | Status         | Content/Comment
+|-----------------------|----------------|---------------------|
+| [kuksa-python-sdk ](https://github.com/eclipse-kuksa/kuksa-python-sdk)| Production | KUKSA Python Client and SDK
+| [kuksa-viss ](https://github.com/eclipse-kuksa/kuksa-viss)| Alpha | VISS2 compatible adapter for KUKSA Databroker
```
[the remark this answers is not in the document]
Comment (erikbosch) [reply profile/README.md:51]: If we actually intend to kill it in the near future we can omit it. If not, I think it should be mentioned, but possibly moved to a "deprecated" section.

[the remark this answers is not in the document]
Comment (SebastianSchildt) [reply profile/README.md:51]: I think this never had much exposure anyway so for this Readme both is fine I guess: Not mentioning it at all, or marking as deprecated.

**Review by SebastianSchildt — APPROVED** [review]
This looks like a good start. lgtm 🏎️

**Pull request merged**